src/cgame/cgame.h

```diff
@@ -602,6 +602,15 @@ typedef struct cg_import_s {
 	 * @param e The entity.
 	 */
 	void (*SetMatrixForEntity)(r_entity_t *e);
+	
+	/**
+	 * @brief Returns the desired tag structure, or `NULL`.
+	 * @param mod The model to check for the specified tag.
+	 * @param frame The frame to fetch the tag on.
+	 * @param name The name of the tag.
+	 * @return The tag structure.
+	 */
+	const r_md3_tag_t *(*GetMeshModelTag)(const r_model_t *mod, int32_t frame, const char *name);
```

Comment:
I *think* most of our "getter" methods in cg_import_t do not actually include "Get" in the name, e.g. `cgi.Cvar`.